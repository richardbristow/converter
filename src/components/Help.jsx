import React from 'react';

const Help = () => {
  return (
    <div>
      This is the help page.
    </div>
  );
};

export default Help;
