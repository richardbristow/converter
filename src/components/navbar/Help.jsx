import React from 'react';

const Help = () => (
  <div>
    <h2>Help</h2>
    This is the help page.
  </div>
);

export default Help;
