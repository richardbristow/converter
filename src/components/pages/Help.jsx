import React from 'react';
import StyledNavPage from '../shared/StyledNavPage';


const Help = () => (
  <StyledNavPage>
    <h2>Help</h2>
    This is the help page.
  </StyledNavPage>
);

export default Help;
