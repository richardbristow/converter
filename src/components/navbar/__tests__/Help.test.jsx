import React from 'react';
import { shallow } from 'enzyme';
import Help from '../Help';

describe('Help', () => {
  it('should render the Help page', () => {
    const wrapper = shallow(<Help />);
    expect(wrapper).toMatchSnapshot();
  });
});
