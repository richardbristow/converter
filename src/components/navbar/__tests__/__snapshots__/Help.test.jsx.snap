// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Help should render the Help page 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Help />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "This is the help page.",
    },
    "ref": null,
    "rendered": "This is the help page.",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "This is the help page.",
      },
      "ref": null,
      "rendered": "This is the help page.",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
